zephyr/module.yml

```diff
@@ -2,3 +2,6 @@ name: nanopb
 build:
   cmake-ext: True
   kconfig-ext: True
+pip:
```

Comment:
here I would call this packages, not pip, so

```
packages:
  python:
   ....

```
This way we can extend this in the future to other things.

Reply:
I think it does make more sense to have the tool itself as a key, Both `packages` and `requirements` are specific to `pip`. It does not make sense to have it for `poetry` for example.

We can add another level to group more package management tools.